db/upgrade.php

```diff
@@ -132,5 +132,15 @@ function xmldb_tool_mfa_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2021021900, 'tool', 'mfa');
     }
 
+    if ($oldversion < 2023031601) {
+        // Update the label field for TOTP to strip tags.
+        $records = $DB->get_records_select('tool_mfa', $DB->sql_like('label', '?'), ['%<%'], '', 'id, label');
+        foreach ($records as $record) {
```

Comment:
This loop could take a long time if we have a lot of users. Anything we could do to speed up? or maybe move to a background?

Reply:
I know it can be longer but this is a setting update for the users who use html tag for devicename (actually just checking `<` though).
I don't think many users are coming to this record update.
I think it's overkill to use background.